x-pack/filebeat/input/awscloudwatch/input.go

```diff
@@ -26,13 +26,21 @@ import (
 	awscommon "github.com/elastic/beats/v7/x-pack/libbeat/common/aws"
 )
 
-const inputName = "awscloudwatch"
+const (
+	inputName    = "aws-cloudwatch"
+	oldInputName = "awscloudwatch"
+)
 
 func init() {
 	err := input.Register(inputName, NewInput)
 	if err != nil {
 		panic(errors.Wrapf(err, "failed to register %v input", inputName))
 	}
+
+	err = input.Register(oldInputName, NewInput)
```

Comment:
Maybe to also log that this name will be deprecated?

Reply:
👍🏼

Reply:
Yep it's in https://github.com/elastic/beats/pull/22228/files#diff-caf6618b123c7fe852f46f717af459389cfc03b351ca89d8263e6500574993e3R93 :) The output log looks like this: 
```
2020-10-28T12:41:21.556-0600    WARN    [aws-cloudwatch]        awscloudwatch/input.go:93       awscloudwatch input name is deprecated, please use aws-cloudwatch instead
```